modules/contact/src/constraints/ComputeDynamicFrictionalForceLMMechanicalContact.C

```diff
@@ -289,7 +289,7 @@ ComputeDynamicFrictionalForceLMMechanicalContact::enforceConstraintOnDof3d(
   }
   else
   {
-    const Real epsilon_sqrt = 1.0e-14;
+    const Real epsilon_sqrt = 1.0e-48;
```

Comment:
Yeah, it is. We are introducing an error by adding this. The goal, as I intended to do it in the first place, was to avoid the MetaPhysicL error. A lower number still avoids this error, so why having it larger.

Reply:
I'm all for making it smaller. Just curious how you arrived at 48 😄

Reply:
The heuristic process was sqrt(1.0e-48) ~ 1.0e-24 which is probably well below the precision required in the contact constraints for a successful run.

Reply:
works for me